avm/res/databricks/workspace/tests/e2e/waf-aligned/dependencies.bicep

```diff
@@ -94,7 +94,8 @@ resource keyPermissions 'Microsoft.Authorization/roleAssignments@2022-04-01' = {
   name: guid('msi-${keyVault::key.id}-${location}-${managedIdentity.id}-Key-Key-Vault-Crypto-User-RoleAssignment')
   scope: keyVault::key
   properties: {
-    principalId: '711330f9-cfad-4b10-a462-d82faa92027d' // AzureDatabricks Enterprise Application Object Id (Note: this is tenant specific)
+    // principalId: '711330f9-cfad-4b10-a462-d82faa92027d' // AzureDatabricks Enterprise Application Object Id (Note: this is tenant specific)
+    principalId: '5167ea7a-355a-466f-ae8b-8ea60f718b35' // AzureDatabricks Enterprise Application Object Id (Note: this is tenant specific)
```

Comment:
I wonder @clintgrove & @eriqua if we should move the value into the `main.test.bicep` - just to make it more obvious.

For example, we could add a parameter `param databricksApplicationObjectId string` to the dependencies.bicep and assign it a value in the `main.test.bicep` where all the other values are configured.

It doesn't matter 'per se' as it works as it is, but it took me 5 retries in my fork until I figured 'Maybe I missing a tenant-specific value in a test file and that's why it fails' 😄

Reply:
HI @AlexanderSehr yes  I have noted this as well. But my thoughts are to NOT add a parameter for this, as if this was a scenario where a customer would use it, they would not have a parameter, as there is only one AppID for databricks in entire tenant, so making a parameter is pointless. I suggest we leave it like it is

Reply:
Agreed offline